src/Inventory/AddList.js

```diff
@@ -54,6 +66,34 @@ export default class AddInventoryList extends Component {
 		});
 	}
 
+	updateList() {
+		if (this.state.name.length === 0) {
+			return Alert.alert(
+				'Error updating Inventory list',
+				'Inventory List name must not be empty',
+				[
+					{ text: "OK", style: "normal" }
+				],
+				{ cancelable: true }
+			)
+		}
+
+		Meteor.call('inventorylists.updateName', this.props.navigation.state.params.id, this.state.name, (err) => {
```

Comment:
Doesn't seem to work for me since we don't have this `inventorylists.updateName` method in the backend

Reply:
Yeah, sorry I should have mentioned to wait for the PR for grocee and then I forgot lol. Will do that

Reply:
Created a PR to add `inventorylists.updateName`: https://github.com/Grocee/grocee/pull/32